generics/helpers/mathFunctions.js

```diff
@@ -1,4 +1,6 @@
-const math = require('mathjs')
+const { create, all } = require('mathjs')
```

Comment:
we can access the create function using math.create why it is specifically imported @praveenKDass